src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+import "./NavSearchResult.scss";
+
+export interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
+}
+
+export const NavSearchResult: React.FC<NavSearchResultProps> = ({
+  title,
+  description,
+  image,
+  isAvatar,
+  onClick,
+}) => {
+  const imageStyles = {
+    backgroundImage: image ? `url(${image})` : undefined,
+  };
```

Comment:
We need to `useMemo`/similar on inline styles like this, as otherwise the object reference changes every time the component function is called.

Reply:
fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/5afa888773470274a321c8b6e1ba1e547b72776c